source/common/listener_manager/listener_impl.cc

```diff
@@ -236,8 +248,10 @@ std::string listenerStatsScope(const envoy::config::listener::v3::Listener& conf
     return absl::StrCat("envoy_internal_", config.name());
   }
   auto address_or_error = Network::Address::resolveProtoAddress(config.address());
-  THROW_IF_NOT_OK_REF(address_or_error.status());
-  return address_or_error.value()->asString();
+  if (address_or_error.status().ok()) {
+    return address_or_error.value()->asString();
+  }
+  return absl::StrCat("invalid_address_listener");
```

Comment:
Is this safe to don't throw or return an error here? And I think the absl::StrCat is unnecessary?

Reply:
yeah this is fine - it basically defers the config load failure until slightly later when we use the invalid address.  arguably slightly better to fast fail but plumbing the error here requires a bunch of work so given the unliklihood of invalid IP address I figured the optimization wasn't worth the refactor.
That said while this is safe it's worth having a regression test, so I'll upload that shortly.